src/docs/development/platform-integration/web.md

```diff
@@ -25,7 +25,9 @@ No.
 
 ### Can I use Flutter plugins?
 
-Not yet.
+Yes, several plugins have web support. 
+Find an updated list of plugins on [pub.dev][] using the web filter.
+You can also add web support to existing plugins or write your own plugins for the web.
```

Comment:
Where can they learn how to do this? I can't remember if we've added docs for this, or if there are any docs on the GitHub wiki.

Reply:
We only have a medium article currently. should i link to that? https://medium.com/flutter/how-to-write-a-flutter-web-plugin-5e26c689ea1

Reply:
Yes, please! That's right. I reviewed that article.